pkgs/applications/networking/cluster/velero/default.nix

```diff
@@ -40,6 +40,6 @@ buildGoModule rec {
       "https://github.com/vmware-tanzu/velero/releases/tag/v${version}";
     license = licenses.asl20;
     maintainers = [ maintainers.mbode maintainers.bryanasdev000 ];
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
This should be platforms.unix.

Reply:
@SuperSandro2000 I will do a PR to refactor these things in my packages, it was my mistake.